File: JS-Basics/For Loop/Exercise/04.js
function solve(age,priceWashingMachine,toyPrice) {
    toysCount = 0
    moneyCount = 10
    money = 0


    for (let i = 1; i <= age; i++) {
        if (i % 2 === 0) {
            money += moneyCount - 1
            moneyCount += 10
        }
        else {
            toysCount += 1
        }
    }

    money += toyPrice * toysCount

    if (priceWashingMachine > money) {
        console.log(`No! ${(priceWashingMachine - money).toFixed(2)}`)
    }
    else {
        console.log(`Yes! ${(money - priceWashingMachine).toFixed(2)}`)
    }
}

File: Basics/For Loop/Exercise/04.py
age = int(input())
washing_machine_price = float(input())
toy_price = int(input())

total_money = 0
toy = 0
money = 0
money_for_brother = 0

for num in range(1, age + 1):
    if num % 2 == 0:
        money_for_brother += 1
        money += 10
        for i in range(10, money + 1, 10):
           total_money += 10
    else:
        toy += 1

sum_money = total_money - money_for_brother
sum_toy = toy * toy_price
total = sum_money + sum_toy

if washing_machine_price > total:
    print(f"No! {(washing_machine_price - total):.2f}")
elif washing_machine_price <= total:
    print(f"Yes! {(total - washing_machine_price):.2f}")
